lib/ace/layer/marker.js

```diff
@@ -181,7 +181,7 @@ var Marker = function(parentEl) {
             "height:", height, "px;",
             "width:", width, "px;",
             "top:", top, "px;",
-            "left:", left, "px;", extraStyle || "", "'></div>"
+            "left:", left, "px;", extraStyle || "", "'>", content || "", "</div>"
```

Comment:
I think it might be better to duplicate this function in easymotion and not make general variant more complicated.
Especially since easymotion doesn't need setting width and have to override that with `!important` in css